examples/README.md

```diff
@@ -0,0 +1,15 @@
+# Lettre Examples
+ This folder contains examples showing how to user lettre in your own projects.
```

Comment:
Didn't catch this earlier (user -> use). Also an empty line under the title makes the markdown better

```suggestion

This folder contains examples showing how to use lettre in your own projects.
```